Gruntfile.js

```diff
@@ -157,17 +157,17 @@ module.exports = function(grunt) {
       }
     },
 
-    'gh-pages': {
```

Comment:
@WellingGuzman Fixed it in https://github.com/h5bp/Effeckt.css/commit/68a51fc9eedf1648300c64366cc679f3fb4979da.